docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -639,3 +639,12 @@ Certain features, such as `--extensions` and `--transformations_config`, are not
 For `--extensions`, the new frontends support only paths to shared libraries (.dll and .so). For `--transformations_config`, they support JSON configurations with defined library fields.
 Inputs freezing (enabled by `--freeze_placeholder_with_value` or `--input` arguments) is not supported on the new frontends.
 The IR conversion falls back to the old path if a user does not select any expected path of conversion explicitly (by `--use_new_frontend` or `--use_legacy_frontend` MO arguments) and unsupported pre-defined scenario is detected on the new frontend path.
+
+#### 106. What do the messages "Output name: `<result_operation_name>` not found" or "Output node with `<result_operation_name>` is not found in graph" mean? <a name="question-106"></a>
```

Comment:
Where do we raise such exception? I don't see it in the MO code.

Reply:
@AlexKoff88, please comment on how FAQ for MO is related to POT and how this error can really appear in MO output.

Reply:
Not in MO output, these errors appear in POT pipeline and come from these lines:

  File "/.../openvino/tools/mo/back/ie_ir_ver_2/emitter.py", line 571, in serialize_network
    assert len(node) == 1, "Output node with name {} is not found in graph.".format(output_name)
AssertionError: Output node with name 109/sink_port_0 is not found in graph. - see 80627 for example;

  File "/.../openvino/tools/accuracy_checker/utils.py", line 997, in postprocess_output_name
    raise ValueError('Output name: {} not found'.format(output_name))
ValueError: Output name: output not found - see 79081 for exaple.

Reply:
If the problem occurs during the quantization using POT (with MO under the hood) we should include this in POT FAQ.

Reply:
> If the problem occurs during the quantization using POT (with MO under the hood) we should include this in POT FAQ.

Where is it placed?

Reply:
If no objections I can move the information from this PR to my own since, anyway, I am working on the revision of POT docs.

Reply:
Yes, I think it will be better.

Reply:
Moved to this [PR](https://github.com/openvinotoolkit/openvino/pull/11072).

Reply:
Thank you! Close this PR.